benchmark_runner/grafana/perf/jsonnet/main.libsonnet

```diff
@@ -286,7 +286,8 @@ g.dashboard.new('PerfCI-Regression-Summary')
           + elasticsearch.withMetrics([
             elasticsearch.metrics.MetricAggregationWithSettings.Max.withField('ci_minutes_time')
             + elasticsearch.metrics.MetricAggregationWithSettings.Max.withId('1')
-            + elasticsearch.metrics.MetricAggregationWithSettings.Max.settings.withScript('Integer.parseInt(\"0\"+doc["ocp_version.keyword"].value.replace(\".\",\"\").replace(\"r\",\"1\").replace(\"e\",\"0\").replace(\"c\",\"\").replace(\"f\",\"\").replace(\"-\",\"\"))')
+            // Convert OCP version string (like 4.20.0-rc.2) to an integer for numeric display in Grafana metrics
+            + elasticsearch.metrics.MetricAggregationWithSettings.Max.settings.withScript('Integer.parseInt(\"0\"+doc[\"ocp_version.keyword\"].value.replace(\".\",\"\").replace(\"r\",\"\").replace(\"c\",\"\").replace(\"e\",\"\").replace(\"f\",\"\").replace(\"-\",\"\") )')
```

Comment:
That looks different from before: you are now replacing `r` with an empty string rather than "1".   Is that intended?